### DISCUSSION TITLE
Update HTML files

### DISCUSSION THREAD

**Discussion by tuvok81** [jomjol/AI-on-the-edge-device#2524] [Q&A]
Hello,
is this possible to update the HTML files on the SDCard without removing it from the ESP?
can i update it via Network?

![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/30367063/c1e69d73-2fd8-4488-8e25-200fe83d6993)

**Comment by friedpa** [comment]
Just use the OTA update function.

**Comment by tuvok81** [comment]
to update the HTML FIles=?

**Comment by friedpa** [comment]
Yes, over the OTA update you can update:
- the whole system 
- the HTML Files
- the Fireware
I would recommend to get the latest official File via the Code -> ZIP Download and update the ESP accordingly

**Comment by tuvok81** [comment]
hm i will not unpack the .zip file than .. is this correct? i do upload the .zip file?

**Comment by friedpa** [comment]
yes, upload the whole ZIP File

**Comment by tuvok81** [comment]
thanks
